security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -124,13 +129,23 @@ func (p *Plugin) useCachedToken() ([]byte, bool) {
 		return nil, false
 	}
 
+	var cacheHitCount uint64 = 0
+	p.mutex.Lock()
+	p.accessTokenCacheHit++
+	cacheHitCount = p.accessTokenCacheHit
```

Comment:
why not just `cacheHitCount = p.accessTokenCacheHit%cacheHitDivisor`.

Reply:
Then we still loose the information about cache hit. The cacheHitCount reflects number of STS requests sent from Envoy.